xbmc/guilib/GUITexture.h

```diff
@@ -56,6 +56,7 @@ class CTextureInfo
   CTextureInfo& operator=(const CTextureInfo& right) = default;
   bool       useLarge;
   CRect      border;          // scaled  - unneeded if we get rid of scale on load
+  bool       infill{true};    // if false, the main body of a texture is not drawn. useful for borders with no inner filling
```

Comment:
should this be `m_infill` for a member object?

Reply:
done.